osgp/platform/osgp-adapter-ws-smartmetering/src/main/java/org/opensmartgridplatform/adapter/ws/smartmetering/endpoints/RequestMessageMetadata.java

```diff
@@ -0,0 +1,106 @@
+/**
+ * Copyright 2016 Smart Society Services B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.opensmartgridplatform.adapter.ws.smartmetering.endpoints;
+
+import java.io.Serializable;
+import lombok.Getter;
+import org.apache.commons.lang3.StringUtils;
+import org.opensmartgridplatform.domain.core.valueobjects.DeviceFunction;
+import org.opensmartgridplatform.shared.infra.jms.MessageType;
+import org.opensmartgridplatform.shared.validation.Identification;
+import org.opensmartgridplatform.shared.wsheaderattribute.priority.MessagePriorityEnum;
+
+@Getter
+public class RequestMessageMetadata {
+
+    @Identification
+    private final String deviceIdentification;
+    @Identification
+    private final String organisationIdentification;
+    private final Serializable requestData;
```

Comment:
I do not think the actual request data should be part of the metadata.

Reply:
you are right